core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/dot-uve-toolbar.component.html

```diff
@@ -1,18 +1,27 @@
 <p-toolbar>
     <div class="p-toolbar-group-start">
-        <p-button icon="pi pi-eye" styleClass="p-button-text" data-testId="uve-toolbar-preview" />
+        @if ($toolbar().editor) {
```

Comment:
Are we creating a second toolbar for preview mode?

Reply:
Yes! Now we have two toolbar
`dot-edit-ema-toolbar`
`dot-uve-toolbar`

The second one is the new toolbar for preview mode